### DISCUSSION TITLE
force app to update

### DISCUSSION THREAD

**Discussion by avishka-codimite** [react-native-tvos/react-native-tvos#475] [Q&A] [answered]
Hi. Can I know if there is a way to force users to update the app when a new version is available google play store or apple app store. 

I found this library when searching about it [react-native-version-check](https://github.com/kimxogus/react-native-version-check).   library working fine on android tv. but there is an issue on tvos side `pod install` not working properly.

error  : **TypeError: undefined is not an object (evaluating 'RNVersionCheck.currentVersion')**

**Comment by douglowder** [comment, marked as answer]
Inspection of https://github.com/kimxogus/react-native-version-check/blob/master/packages/react-native-version-check/react-native-version-check.podspec shows that this package is currently configured only for iOS platform in Cocoapods. You would need to remove the `s.platform` line and add this instead:

```podspec
s.platforms              = { :ios => "12.4", :tvos => "12.4" }
```
